budgetportal/openspending.py

```diff
@@ -121,6 +121,17 @@ def aggregate(self, cuts=None, drilldowns=None):
             raise Exception("More cells than expected - perhaps we should start paging")
         return aggregate_result.json()
 
+    def filter_dept(self, result, dept_name, sphere, financial_year):
+        sphere_filter = {
+            'national': 'vote_number.department',
```

Comment:
I think these names are coincidental - I think if we use self.get_department_dimension() or something like that we can keep writing code using OpenSpending types instead of column names.